### PR TITLE
stdlib: remove _Reflectable

### PR DESCRIPTION
**Pull request by gribozavr**
Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| All supported platforms | @swift-ci Please smoke test and merge |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

**Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| All supported platforms | @swift-ci Please test and merge |
| OS X platform | @swift-ci Please test OS X platform |
| OS X platform | @swift-ci Please benchmark |
| Linux platform | @swift-ci Please test Linux platform |

**Lint Testing**

| Language | Comment |
| --- | --- |
| Python | @swift-ci Please Python lint |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by gribozavr** [comment]
@swift-ci Please benchmark

**Comment by gribozavr** [comment]
@swift-ci Please benchmark

**Comment by gribozavr** [comment]
@swift-ci Please benchmark

**Comment by gribozavr** [comment]
@swift-ci Please test

File: stdlib/public/runtime/Reflection.mm [hunk 1]
```diff
@@ -313,71 +304,25 @@ void swift_MagicMirrorData_summary(const Metadata *T, String *result) {
   return swift_getObjCClassMetadata(isa);
 }
  
-static std::tuple<const _ReflectableWitnessTable *, const Metadata *,
-                  const OpaqueValue *>
+static std::tuple<const Metadata *, const OpaqueValue *>
 getReflectableConformance(const Metadata *T, const OpaqueValue *Value) {
```
Comment (slavapestov) [inline stdlib/public/runtime/Reflection.mm]: The name of this function no longer makes sense. How about just unwrapExistential()?

**Comment by slavapestov** [comment]
Looks good, other than potentially renaming the function.

**Comment by gribozavr** [comment]
@slavapestov Thank you for the review, I renamed it.

**Comment by gribozavr** [comment]
@swift-ci Please test and merge

**Pull request merged**